src/common/types/metadataQueries.js

```diff
@@ -18,6 +18,7 @@ type MetadataQueryOrderByClause = {
 
 type MetadataQuery = {
     ancestor_folder_id: string,
+    fields?: string[],
```

Comment:
Alternatively you could write this as `Array<string>` which I see used in BUIE often